### DISCUSSION TITLE
What is the account on Haier hOn?

### DISCUSSION THREAD

**Discussion by Look-up-the-sky** [Andre0512/hon#87] [Q&A]
After installation，it need an account number and password. What is the account？I try haier's account，but it doesn't work.
![image](https://github.com/Andre0512/hon/assets/50399445/424005e0-d337-441f-b5e4-1b9a2bcd4d21)

**Comment by skddyj** [comment]
i have the same question

**Comment by Andre0512** [comment]
It's the same data you use in the [hOn app](https://hon-smarthome.com/). 
Check if you really using the haier hOn app, see #37.
